budou/budou.py

```diff
@@ -58,9 +58,16 @@ def main():
   if args['--version']:
     print(__version__)
     sys.exit()
+    
+  if args['<source>']:
+    source = args['<source>']
+  elif not sys.stdin.isatty():
+   source = sys.stdin.read()
+  else:
+    sys.exit()
 
```

Comment:
Could you add another logic to show the command help if the source is empty?

Reply:
Done